client/config/environment.js

```diff
@@ -4,12 +4,14 @@ var devHost = 'http://localhost:1337';
 var prodHost = 'http://gallery.hairbrain.io';
 var devS3 = 'https://gallery13-dev.s3.amazonaws.com';
 var prodS3 = 'https://gallery13.s3.amazonaws.com';
+var gaId = 'UA-12392326-1';
 
 module.exports = function(environment) {
   var ENV = {
     modulePrefix: 'client',
     podModulePrefix: 'client/pods',
     environment: environment,
+    isProduction: (this.environment === 'production'),
```

Comment:
I'm still not convinced this is the best solution, it feels a bit like a workaround.  Couldn't you set the Google settings directly within https://github.com/hairbrain/gallery13/blob/fix_56/client/config/environment.js#L31 and https://github.com/hairbrain/gallery13/blob/fix_56/client/config/environment.js#L63?

Then, instead of testing for environment within the brocfile, you could just look at the google analytics settings?  As it is now, there seem to be a lot of duplicate checks.